### ISSUE TITLE
Handsontable integration

### ISSUE THREAD

**Issue by doczoidberg** [handsontable/hyperformula#560]
can you give a rough estimation when hf will work in (beta) with handsontable?

**Comment by wojciechczerniak** [comment]
Unfortunately, no. We're still missing 130+ functions before we can release HyperFormula with Handsontable. Until we're close to that milestone the integration is on hold.
